checklist/app/views/todos/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h2> Create New Task </h2>
+
+<%= form_for @todo do |f| %>
+  <p>
+  <%= f.label :content %><br>
+  <%= f.text_field :content %>
+</p>
+
+<p>
+  <%= f.label "Is completed True or False?"  %><br>
+  <%= f.text_field :is_completed %>
+</p>
+
+
+  <%= f.submit "Create" %>
+<% end %>
+
+<h4> <a href="/todos"> < Back </a></h4>
```

Comment:
`<h4> <a href="/todos"> < Back </a></h4>`

You have an extra `<` tag in here but, otherwise, this looks good. 👍